cmd/fleetctl/apple_mdm.go

```diff
@@ -44,124 +52,137 @@ func appleMDMCommand() *cli.Command {
 	}
 }
 
-func appleMDMSetupCommand() *cli.Command {
+func generateCommand() *cli.Command {
 	return &cli.Command{
-		Name:  "setup",
-		Usage: "Setup commands for Apple MDM",
+		Name: "generate",
```

Comment:
request from product: hide all MDM commands until we officially release this feature.

```suggestion
		Name: "generate",
		Hidden:  true, // TODO: temporary, until the MDM feature is officially released
```

Reply:
done